### PR TITLE
PXC 3960 Release notes and release note tasks for 5.7.38-31.59 (5.7)

### PR DESCRIPTION
**Pull request by patrickbirch**
A  source/_res/5.7-useful-links.txt
A  source/_res/5.7.38-31.59-rh.txt
A  source/_res/pxc-blurb.txt
M source/conf.py
A  source/release-notes/5.7.38-31.59.rst
M  source/release-notes/release-notes_index.rst

### CODE REVIEW

File: doc/source/_res/5.7.38-31.59-rh.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Release Highlights
+=================================================
+
+The following lists a number of the notable updates and fixes for MySQL 5.7.38, provided by Oracle, and included in Percona Server for MySQL:
```
Comment (alina-derkach-oaza) [inline doc/source/_res/5.7.38-31.59-rh.txt]: Improvements and bug fixes introduced by Oracle for MySQL 8.0.28 and included in Percona Server for MySQL are the following:

Comment (george-lorch) [reply doc/source/_res/5.7.38-31.59-rh.txt]: This is for 5.7.38, not 8.0.28 ;)

**Review by george-lorch — APPROVED** [review]

File: doc/source/_res/5.7.38-31.59-rh.txt (hunk 1 above)
Comment (patrickbirch) [reply doc/source/_res/5.7.38-31.59-rh.txt]: This is 5.7.38.

Comment (alina-derkach-oaza) [reply doc/source/_res/5.7.38-31.59-rh.txt]: Yeah, sorry. Then -> Improvements and bug fixes introduced by Oracle for MySQL 5.7.38 and included in Percona Server for MySQL are the following:

**Review by alina-derkach-oaza — APPROVED** [review]

**Pull request merged**